### PR TITLE
Create Bobby gosselin huntsville recommendation file

### PR DESCRIPTION
**Pull request by bobby-gosselin**
- This is a bullet point.

- [ ] This is a task list item.

### CODE REVIEW

**Comment by othersideoflearning** [comment]
Good job

**Comment by jeremypollack** [comment]
Hello!  I am leaving a general comment!  Who is this General Comment and why is he looking at your project?

File: bobby_gosselin-Huntsville.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Huntsville
+
+### Places to Eat
+- Bandito Burrito
+- Guadalhara
+- Pieology
```
Comment (jeremypollack) [inline bobby_gosselin-Huntsville.md]: mmmmm, pie!

**Review by jeremypollack — APPROVED** [review]
Great work!

**Pull request merged**